tests/data/results_file.yaml.new

```diff
@@ -0,0 +1,3 @@
+test_pipe:
```

Comment:
This file shouldn't be committed to the repo, should it?

Reply:
Removed along with the ".new" usage.